querybook/server/lib/github/github.py

```diff
@@ -38,34 +44,89 @@ def oauth_config(self) -> Dict[str, Any]:
         return config
 
     def save_github_token(self, token: str) -> None:
-        flask_session[GITHUB_ACCESS_TOKEN] = token
-        LOG.debug("Saved GitHub token to session")
+        encrypted_token = encrypt_token(token)
+        user = get_user_by_id(current_user.id)
+        if user:
+            try:
+                update_user_properties(user.id, github_access_token=encrypted_token)
+                LOG.debug("Saved encrypted GitHub token to user properties")
+            except Exception as e:
+                LOG.error(f"Failed to update user properties: {e}")
+                raise AuthenticationError("Failed to save GitHub token")
+        else:
+            LOG.error("User not found when saving GitHub token")
+            raise AuthenticationError("User not found")
 
-    def get_github_token(self) -> Optional[str]:
-        access_token = flask_session.get(GITHUB_ACCESS_TOKEN)
-        if not access_token:
-            LOG.error("GitHub OAuth token not found in session")
-            raise Exception("GitHub OAuth token not found in session")
-        return access_token
+    def get_github_token(self) -> str:
```

Comment:
other than the crypto utils above, I think we can also generalize those token related operations, by given the user id and the token type